doc/Vagrant.md

```diff
@@ -10,27 +10,26 @@ Getting started
 No need to setup up a webserver, database etc. before actually starting.
 Vagrant creates a virtual machine for you that you can just run inside VirtualBox and start to work directly on Friendica.
 
-It brings an Ubuntu Xenial (16.04) with PHP 7.0 and MySQL 5.7.16
+It brings an Debian Bullseye with PHP 7.4 and MariaDB 10.5.11.
 
 What you need to do:
 
 1. Install VirtualBox and vagrant.
 Please use an up-to-date vagrant version from https://www.vagrantup.com/downloads.html.
 2. Git clone your Friendica repository.
-Inside, you'll find a "Vagrantfile" and some scripts in the utils folder.
-3. Run "vagrant up" from inside the friendica clone:
-        $> vagrant up
-Be patient: When it runs for the first time, it downloads an Ubuntu Server image.
-4. Run "vagrant ssh" to log into the virtual machine to log in to the VM:
-        $> vagrant ssh
+Inside, you'll find a `Vagrantfile` and some scripts in the `bin/dev` folder.
```

Comment:
As `composer install `is not run anymore in provisioning script, there should be a step between `git clone` and `vagrant up` which instructs to run `composer install`